src/main/java/edu/kis/powp/jobs2d/command/visitor/DriverCommandTransformVisitor.java

```diff
@@ -0,0 +1,73 @@
+package edu.kis.powp.jobs2d.command.visitor;
+
+import edu.kis.powp.jobs2d.command.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DriverCommandTransformVisitor implements DriverCommandVisitor {
+    private double scaleX = 1.0;
+    private double scaleY = 1.0;
+    private double translateX = 0.0;
+    private double translateY = 0.0;
+    private double rotation = 0.0;
+
+    private final List<DriverCommand> transformedCommands = new ArrayList<>();
+
+    public List<DriverCommand> getTransformedCommands() {
+        return transformedCommands;
+    }
+
+    private int[] apply(int x, int y) {
+        double newX = x * scaleX;
+        double newY = y * scaleY;
+
+        double radians = Math.toRadians(rotation);
+        double rotatedX = newX * Math.cos(radians) - newY * Math.sin(radians);
+        double rotatedY = newX * Math.sin(radians) + newY * Math.cos(radians);
+
+        rotatedX += translateX;
+        rotatedY += translateY;
```

Comment:
Lack of SRP - many transformations in one class.
and OCP - the definition of transformations has a fixed order and cannot be extended without modifications.

Please reuse (and extend with translate operation) transformations from PR #9.